@kalbasit great idea, I added a new commit. I tried to test this locally by running

    nix-env -i go -f .

(at the root of this repo)

but I get this error:

```
--- FAIL: TestChown (0.00s)
        os_unix_test.go:51: gid: 100
        os_unix_test.go:63: groups:  [65534 100]
        os_unix_test.go:66: chown /tmp/_Go_TestChown141509686 -1 65534: chown /tmp/_Go_TestChown141509686: invalid argument
FAIL

```

I'm on Fedora 30 5.3.14-200.fc30.x86_64 

Thanks!

--- a/pkgs/development/compilers/go/1.13.nix
+++ b/pkgs/development/compilers/go/1.13.nix
@@ -57,8 +57,8 @@ stdenv.mkDerivation rec {
       --replace '#!/usr/bin/env bash' '#!${runtimeShell}'
 
     # Patch the mimetype database location which is missing on NixOS.
-    substituteInPlace src/mime/type_unix.go \
-      --replace '/etc/mime.types' '${mailcap}/etc/mime.types'
+    # but also allow static binaries built with NixOS to run outside nix
+    sed -i 's,\"/etc/mime.types,"${mailcap}/etc/mime.types\"\,\n\t&,' src/mime/type_unix.go
 
     # Disabling the 'os/http/net' tests (they want files not available in
     # chroot builds)

